src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpTransportContext.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Security.Authentication.ExtendedProtection;
+
+using SafeInternetHandle = Interop.WinHttp.SafeInternetHandle;
+
+namespace System.Net.Http
+{
+    internal class WinHttpTransportContext : TransportContext
+    {
+        private WinHttpChannelBinding _channelBinding = null;
```

Comment:
Apparently the FxCop rule that got drilled into my brain on this point was removed after .NET 2.0.  The Internet seems to suggest that optimizations in the VS2010 era removed the redundant statements from the initializer.

I feel like I've seen F10 walk over those lines, though... probably a Retail-only optimization.  Oh, well, I'm going to go feel old now for parroting a rule that has been not a rule for 5 years.